api/jobs.go

```diff
@@ -3,6 +3,7 @@ package api
 import (
 	"context"
 	"encoding/json"
+	"github.com/ONSdigital/dp-search-reindex-api/pagination"
```

Comment:
explicit import not needed, I think

Reply:
it does seem to be needed but it's not in the right place so good spot - I will change it

Reply:
done